var/spack/repos/builtin/packages/qhull/package.py

```diff
@@ -36,16 +37,13 @@ class Qhull(Package):
 
     homepage = "http://www.qhull.org"
 
-    version('7.2.0', 'e6270733a826a6a7c32b796e005ec3dc',
+    version('2015.2', 'e6270733a826a6a7c32b796e005ec3dc',
             url="http://www.qhull.org/download/qhull-2015-src-7.2.0.tgz")
 
-    version('1.0', 'd0f978c0d8dfb2e919caefa56ea2953c',
+    version('2012.1', 'd0f978c0d8dfb2e919caefa56ea2953c',
             url="http://www.qhull.org/download/qhull-2012.1-src.tgz")
```

Comment:
Although each version actually has a build number (1.0 should be 6.3.1.1494), most packages refer to the qhull version by its year. This includes the [qhull downloads page](http://www.qhull.org/download/).